<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CoreSlamPF_HMI</class>
 <widget class="QMainWindow" name="C_CoreSlamPF_HMI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>483</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>483</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CoreSlam Particles Filter Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_coreSlamFrameRate">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamFrameRate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gv_Map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CenterMap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveRightMap">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveLeftMap">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveUpMap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomInMap">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MoveDownMap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomDefaultMap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ZoomOutMap">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ResetSlam">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset SLAM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>World map</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamErrX">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamErrX</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamErrY">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamErrY</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamErrAngle">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamErrAngle</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamPosY">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamPosY</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamPosAngle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamPosAngle</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamPosX">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamPosX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkB_FollowRobot">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot Tracking</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamPosQuality">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamPosQuality</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_coreSlamEstimatorIndex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lbl_coreSlamEstimatorIndex</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
